stan/math/prim/prob/bernoulli_logit_lpmf.hpp

```diff
@@ -47,7 +47,7 @@ return_type_t<T_prob> bernoulli_logit_lpmf(const T_n& n, const T_prob& theta) {
   }
   T_n_ref n_ref = n;
   T_theta_ref theta_ref = theta;
-  check_bounded(function, "n", n_ref, 0, 1);
+  check_bounded(function, "n", value_of(n_ref), 0, 1);
```

Comment:
Integer, `value_of` not necessary

Reply:
Is it guranteed somewhere to always be an integer or do we only check that at the Stan language level?

Reply:
Stan language. These are the signatures:

```stan
real bernoulli_logit_lpmf(int, real)
real bernoulli_logit_lpmf(int, vector)
real bernoulli_logit_lpmf(int, row_vector)
real bernoulli_logit_lpmf(int, real[])
real bernoulli_logit_lpmf(int[], real)
real bernoulli_logit_lpmf(int[], vector)
real bernoulli_logit_lpmf(int[], row_vector)
real bernoulli_logit_lpmf(int[], real[])
```

Reply:
idk, idt there's anything bad about calling value_of here. Idt it would be an error and I don't think it would be a performance thing